.gitignore

```diff
@@ -68,3 +68,6 @@ tmp/
 package-lock.json
 *.bak
 t.*
+
+# Vscoded setting
+.vscode
```

Comment:
`.vscode/setting.json` should not be ignored...

Reply:
My vscode always change `"python.pythonPath"` automatically.

Anyway, I will cancel that ignore.